about.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Lato" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d334bd1716.js" crossorigin="anonymous"></script>
+  <!-- Bootstrap CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/about.css">
+    <title>About Kadoma Community Development</title>
+</head>
+<body>
+    <nav>
+        <ul class="d-lg-flex d-none justify-content-end sp-bg list-unstyled">
+            <li class="px-1 py-1">
+                <a href="#"><i class="fa fa-facebook"></i> </a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">Logout</a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">My Page </a>
+            </li>
+            <li class="px-2 py-1">
+                <a href="#">English</a>
+            </li>
+        </ul>
+    </nav>
+    <nav class="navbar navbar-expand-lg">
+        <div class="container">
+            <a class="navbar-brand d-none d-md-flex" href="./index.html"><img class="conta" src="img/work.png" alt="logo"> <h1>Kadoma Community Development</h1></a>
+            <div class="hamburger d-lg-none">
+                <img class="ham" src="img/ham2.png" alt="hamburger-icon">
+            </div>
+            <div class="d-none d-lg-block justify-content-end" id="navbarNav">
+                <div class="d-flex align-items-center justify-content-around">
+                    <a class="navbar-brand d-lg-none" href="./index.html"><img src="img/fbc.png" alt="logo"><h1>Kadoma <br>Community Development</h1></a>
```

Comment:
* The menu bar is differnt from the original design kindly 🙏 compare and make the neccesaryy adjustments 
* And also the page title section try to stick to the design as much as possible as they look different 🙏 
* YOURS : 
![image](https://user-images.githubusercontent.com/52497246/133553230-5b9d5fdc-4ced-4b3f-9cc0-dbbfce0e884b.png)
* ORIGINAL : 
![image](https://user-images.githubusercontent.com/52497246/133553289-e75a6c67-3462-4e81-aab2-8073deccc5b6.png)